README.md

```diff
@@ -30,14 +30,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The .forEach method helps us retrieve information about a given array; whereas, .map allows us to make changes to items in an array, and returns a new array.
```

Comment:
You can use "forEach" to modify an existing array as well as retrieve information.